mockgcp/mockresourcemanager/normalize.go

```diff
@@ -24,4 +28,42 @@ func (s *MockService) ConfigureVisitor(url string, replacements mockgcpregistry.
 }
 
 func (s *MockService) Previsit(event mockgcpregistry.Event, replacements mockgcpregistry.NormalizingVisitor) {
+	if !isCloudResourceManagerAPI(event) {
+		return
+
+	}
+	name := ""
+	event.VisitResponseStringValues(func(path string, value string) {
+		if path == ".name" {
+			name = value
+		}
+	})
+
+	tokens := strings.Split(name, "/")
+	if len(tokens) == 2 && tokens[0] == "tagKeys" {
+		if name == "namespaced" {
+			// This is actually a search operation: https://cloud.google.com/resource-manager/reference/rest/v3/tagKeys/getNamespaced
+		} else {
+			replacements.ReplaceStringValue(tokens[1], "${tagKeyID}")
+		}
+	}
+	if len(tokens) == 2 && tokens[0] == "tagValues" {
+		replacements.ReplaceStringValue(tokens[1], "${tagValueID}")
+	}
+	if len(tokens) == 2 && tokens[0] == "tagBindings" {
```

Comment:
Nit: From the corresponding gcloud documentation
```
        $ gcloud resource-manager tags bindings create \
            --tag-value=789/env/test \
            --parent=//cloudresourcemanager.googleapis.com/projects/1234
```
Wondering if this means we will need to support more than 2 tokens?